js/unit_test_runner.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+// TODO(bartlomieju): enable linter
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
+import "./unit_tests.ts";
+
+import { permissionCombinations } from "./test_util.ts";
```

Comment:
Is this all combinations of permissions ? I guess many combinations do not have tests. Ideally you’d get a list of all the tests (by importing js/unit_tests.ts and perhaps adding something to testing to extract all registered tests) and from that determine which permissions combinations are needed.

Reply:
That's exactly what this does. Check out `registerPermCombination`

Reply:
Ah cool!